storage/ledger.go

```diff
@@ -9,7 +9,7 @@ import (
 // we aim to store a state of the order of 10^10 registers with up to 1M historic state versions
 type Ledger interface {
 	module.ReadyDoneAware
-	EmptyStateCommitment() flow.StateCommitment
+	DummyStateCommitment() flow.StateCommitment
```

Comment:
I think this should still be
```suggestion
	EmptyStateCommitment() flow.StateCommitment
```
We want the surrounding code to be able to test for an empty ledger.

Reply:
Good catch! thank you 🙏  This was updated as I did a "replace all" for `EmptyStateCommitment`.